nodejs/week2/src/backend/routes/reservations.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const router = express.Router()
+const ReservationsArray = require('../data/reservations.json');
+
+
+
+router.get('/reservations', (req, res)=>{
+        
+// ===================================================================================  for id :  /reservations?id=2
+      const givenid = req.query.id;

```

Comment:
According to the homework description, it should be part of `params`, not part of `query`. The same comment is applicable to all the following routers.

Reply:
I am having some trouble to do that, i will appreciate if you can help me with this one. Thanks 🙂